lib/Alchemy/Phrasea/Controller/Root/AccountController.php

```diff
@@ -299,13 +306,102 @@ public function displayAccount()
         $manager = $this->getEventManager();
         $user = $this->getAuthenticatedUser();
 
+        $repo_baskets = $this->getBasketRepository();
+        $baskets = $repo_baskets->findActiveValidationAndBasketByUser($user);
+
+        $apiAccounts = $this->getApiAccountRepository()->findByUser($user);
+
+        $ownerPublisher = $this->getFeedPublisherRepository()->findBy(['user' => $user, 'owner' => true]);
+
+        $ownerFeedbackSend = $this->getValidationSessionRepository()->findby(['initiator' => $user, ]);
+
         return $this->render('account/account.html.twig', [
-            'user'          => $user,
-            'evt_mngr'      => $manager,
-            'notifications' => $manager->list_notifications_available($user),
+            'user'                => $user,
+            'evt_mngr'            => $manager,
+            'notifications'       => $manager->list_notifications_available($user),
+            'baskets'             => $baskets,
+            'api_accounts'        => $apiAccounts,
+            'isOwnerPublisher'    => count($ownerPublisher) > 0 ? true : false,
+            'isOwnerFeedbackSend' => count($ownerFeedbackSend) > 0 ? true : false,
```

Comment:
For previous args ($user, $manager, ...), the whole object is passed to twig which is responsible to use the object.
It would be better to do the same : pass $ownedFeeds, $initiatedValidations, and let twig do the if(count(...)), because if gives you the chance to display things like "you own X feeds", not only a bool.

btw : comparison ops (<,>,==,<=,...) are booleans, so there is no need to add ternary.

Reply:
ok